src/main.c

```diff
@@ -49,6 +52,61 @@ static void softReset_button_callback(struct device *port, struct gpio_callback
 	soft_reboot();
 }
 
+
+void cdc_acm_descriptor_callback (cdc_acm_cfg_t *cfg)
+{
+	customer_data_t *otp_data = (customer_data_t *)CUSTOMER_DATA_ADDRESS;
+
+	memset(cfg->vendor_string, '\0', 32);
```

Comment:
@bigdinotech can you remove these memset calls? after each `memcpy` below, you can just do (for example)

    cfg->vendor_string[otp_data->vendor_name_len] = '\0';